serpy/serializer.py

```diff
@@ -83,16 +123,19 @@ class FooSerializer(Serializer):
     """
     #: The default getter used if :meth:`Field.as_getter` returns None.
     default_getter = operator.attrgetter
+    default_setter = attrsetter
 
-    def __init__(self, obj=None, many=False, **kwargs):
+    def __init__(self, obj=None, data=None, many=False, **kwargs):
```

Comment:
what do you think about allowing an instance of a serializer to either serialize or deserialize, but not both. This could have one obj parameter like it was before, and it will either fill out self._representation or self._internal_value, and throw an exception if it tries to do both.